src/config/dynamic-theme-fixes.config

```diff
@@ -2486,6 +2486,15 @@ INVERT
 
 ================================
 
+jpl.nasa.gov
+
+CSS
+.brand_area {
```

Comment:
Change it to

```
IGNORE IMAGE ANALYSIS
.brand_area
```
Should do the same affect but cleaner.

Reply:
I already tested it and `IGNORE IMAGE ANALYSIS` makes the text black on black. There are 2 provided images with the logo, one with black text and one with white text. When scrolling the logo moves from ontop of an image to a black background. Ignoring the analysis keeps black text on a dark background. Manually setting it to white retains the logo color and white text.